src/main/java/org/folio/linked/data/integration/ResourceModificationEventListener.java

```diff
@@ -74,7 +75,13 @@ private void sendToSearch(Resource resource) {
   }
 
   private void sendToInventory(Resource resource) {
-    extractInstances(resource).forEach(kafkaInventorySender::sendInstanceCreated);
+    extractInstances(resource)
+      .stream()
+      .filter(this::isNotSourcedFromSrs)
+      .forEach(kafkaInventorySender::sendInstanceCreated);
   }
 
+  private boolean isNotSourcedFromSrs(Resource resource) {
+    return isNull(resource.getInventoryId()) && isNull(resource.getSrsId());
```

Comment:
I don't think this is safe approach. 
Let's say we implement whole sync cycle LinkedData <-> Inventory/SRS.
Thus, an entity, initially created in Linked Data editor, will be sent to Inventory where inv/srs ids will be assigned to it and send back to us. And we will ignore such update, except inv/srs ids, which should be added to our initial entity, right? This is required to make the data consistent in Folio as a system.

So, I suppose we better add a new column/field to resource table/entity, the "Source"

Reply:
Refactored to use "source"